test/Benchmarks/OrleansBenchmarkGrainInterfaces/MapReduce/DataflowGrainsInterfaces.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Orleans;
+
+namespace OrleansGrainInterfaces.MapReduce
+{
+    public interface IDataflowGrain : IGrainWithGuidKey
+    {
+        Task Complete();
+
+        Task Fault();
+
+        Task Completion();
+    }
+
+    public interface ITargetGrain<in TInput> : IDataflowGrain
```

Comment:
Plus, it's purpose in being generic - new dataflow related tests won't need another <code>ITargetGrain</code>